src/flow/io/ical.js

```diff
@@ -1,12 +1,14 @@
 import { parse, Component, Event } from 'ical.js'
 import { createEvents } from 'ics'
+import moment from 'moment'
 
 function eventFromIcal(d) {
   var vevent = new Event(d)
   return {
     title: vevent.summary,
     start: vevent.startDate.toJSDate().toISOString(),
     end: vevent.endDate.toJSDate().toISOString(),
+    duration: vevent.duration ? vevent.toJSDate().toTimeString() : null,
```

Comment:
this should be vevent **.duration.** toSomestring

Reply:
Done.